### PR TITLE
[toolbars-ui] fix dropdown button selection state

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

**Comment by danarish4** [comment]
I find it strange that alignment modals (for example) are registered as `DropdownModal`.
For each standard button (e.g. alignmentPanel, sizePanel, which will also have options as props), I think we should create a unique dropdown. That way, alignmentButton will register alignmentPanel as its modal.
By doing so, alignmentButton will be a simple button that opens alignmentPanel when clicked.
In addition, DropdownButton component will no longer be required.

File: packages/plugin-audio/web/src/getToolbarButtons.tsx [hunk 1]
```diff
@@ -34,15 +34,27 @@ export const getToolbarButtons = (config, services): ToolbarButton[] => {
   return [
     {
       id: PLUGIN_TOOLBAR_BUTTON_ID.SIZE,
+      modal: {
+        Component: DropdownModal,
```
Comment (danarish4) [inline packages/plugin-audio/web/src/getToolbarButtons.tsx:38]:
```suggestion
        Component: decorateComponentWithProps(SizePanel, { options:{['SMALL', 'CONTENT'] as PluginContainerData_Width_Type[]} }),
```

File: packages/plugin-audio/web/src/getToolbarButtons.tsx [hunk 2]
```diff
@@ -34,15 +34,27 @@ export const getToolbarButtons = (config, services): ToolbarButton[] => {
...
+        id: AUDIO_BUTTONS.size,
+      },
       renderer: toolbarItem => (
         <NodeSizeButton
+          id={AUDIO_BUTTONS.size}
           toolbarItem={toolbarItem}
           options={['SMALL', 'CONTENT'] as PluginContainerData_Width_Type[]}
```
Comment (danarish4) [inline packages/plugin-audio/web/src/getToolbarButtons.tsx:45]:
```suggestion
```

File: packages/plugin-audio/web/src/getToolbarButtons.tsx [hunk 3]
```diff
@@ -34,15 +34,27 @@ export const getToolbarButtons = (config, services): ToolbarButton[] => {
...
         />
       ),
     },
     {
       id: PLUGIN_TOOLBAR_BUTTON_ID.ALIGNMENT,
+      modal: {
+        Component: DropdownModal,
```
Comment (danarish4) [inline packages/plugin-audio/web/src/getToolbarButtons.tsx:52]:
```suggestion
        Component: AlignmentPanel,
```

File: packages/plugin-button/web/src/getToolbarButtons.tsx [hunk 4]
```diff
@@ -1,15 +1,24 @@
...
+import { DropdownModal, NodeAlignmentButton } from 'wix-rich-content-toolbars-ui';
+import { buttonsModals, BUTTON_BUTTONS } from './constants';
 import ButtonSettingsModal from './modals/SettingsModal';
 
 export const getToolbarButtons = (config, services, type): ToolbarButton[] => {
   return [
     {
       id: PLUGIN_TOOLBAR_BUTTON_ID.ALIGNMENT,
+      modal: {
+        Component: DropdownModal,
```
Comment (danarish4) [inline packages/plugin-button/web/src/getToolbarButtons.tsx:16]:
```suggestion
        Component: AlignmentPanel,
```

**Comment by tamrat-bay** [comment]
> I find it strange that alignment modals (for example) are registered as `DropdownModal`. For each standard button (e.g. alignmentPanel, sizePanel, which will also have options as props), I think we should create a unique dropdown. That way, alignmentButton will register alignmentPanel as its modal. By doing so, alignmentButton will be a simple button that opens alignmentPanel when clicked. In addition, DropdownButton component will no longer be required.

@danarish4 
As I understand it, you would like different Dropdown components
For each button ( e.g. AlignmentPanel / SizePanel )  instead of using `DropdownModal.tsx ` is it right?
If so, we will have different Dropdown components, but the code will be duplicated

File: e2e/tests/plugins.e2e.ts [hunk 5]
```diff
@@ -95,7 +95,7 @@ import { getTestPrefix } from '../cypress/utils';
     });
 
     context('divider', () => {
-      it(`${getTestPrefix(useTiptap)} render plugin toolbar and change styling`, function () {
+      it.only(`${getTestPrefix(useTiptap)} render plugin toolbar and change styling`, function () {
```
Comment (danarish4) [inline e2e/tests/plugins.e2e.ts:98]:
```suggestion
      it(`${getTestPrefix(useTiptap)} render plugin toolbar and change styling`, function () {
```

**Review by danarish4 — APPROVED** [review]
Looks good!

**Pull request merged**